app/src/main.ts

```diff
@@ -26,7 +26,7 @@ Vue.config.devtools = true
  */
 async function startVue () {
   // Fetch the configuration
-  const envConfig = await getConfig()
+  const envConfig = getConfig()
```

Comment:
This is no longer async (which dated from the time when it fetched the config from the webserver).